sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -129,11 +129,25 @@ def _create_environment(options):
               env=(config.get('env') or '')
           ).SerializeToString())
     elif environment_urn == common_urns.environments.EXTERNAL.urn:
+      def looks_like_json(environment_config):
+        import re
+        return re.match(r'\s*\{.*\}\s*$', environment_config)
+
+      if looks_like_json(portable_options.environment_config):
+        config = json.loads(portable_options.environment_config)
+        url = config.pop('url', None)
```

Comment:
```suggestion
        url = config.get('url', None)
```

Why pop?

Reply:
Thanks Maximilian! I've updated in this commit: https://github.com/apache/beam/pull/9730/commits/95785590993a1035e4619e0f2b7a4434e03f1abc